
public class Exception12 {
public static void main(String args[])
{
	try
	{
		int data=10/0;
		System.out.println("restis the code");
	}
catch(ArithmeticException e)
	{
	System.out.println(e);
	}
}}


